### PR TITLE
Feature istio adapter log format

### PR DESCRIPTION
**Pull request by champly**
### Issues associated with this PR

#1183 

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/log/accesslog.go [hunk 1]
```diff
@@ -152,11 +153,14 @@ func parseFormat(format string) ([]*logEntry, error) {
 					}
 
 					// var def ends, add variable
-					_, err := variable.AddVariable(format[lastMark+1 : pos])
+					varName := format[lastMark+1 : pos]
+					_, err := variable.AddVariable(varName)
 					if err != nil {
-						return nil, err
+						// adapte istio unknow fields
```
Comment (wangfakang) [inline pkg/log/accesslog.go:159]: The format support this style `"log_format": "%StartTime% %RequestReceivedDuration% %ResponseReceivedDuration% %REQ.requestid% %REQ.cmdcode% %RESP.requestid% %RESP.service%"`.

File: pkg/proxy/var.go [hunk 2]
```diff
@@ -208,6 +211,23 @@ func upstreamHostGetter(ctx context.Context, value *variable.IndexedValue, data
...
+}
+
+func upstreamClusterGetter(ctx context.Context, value *variable.IndexedValue, data interface{}) (string, error) {
+	proxyBuffers := proxyBuffersByContext(ctx)
+	stream := proxyBuffers.stream
+
+	if stream.cluster != nil {
+		return stream.cluster.Name(), nil
+	}
+	return variable.ValueNotFound, nil
```
Comment (wangfakang) [inline pkg/proxy/var.go:228]: It should be create a specific error when the variable not found.

**Review by wangfakang — APPROVED** [review]
LGTM.

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Pull request merged**